sdk/core/azure-core/src/main/java/com/azure/core/implementation/util/FluxUtil.java

```diff
@@ -178,11 +178,31 @@ private static void accept(ByteArrayOutputStream byteOutputStream, ByteBuffer by
      * @return The azure context
      */
     private static Context toAzureContext(reactor.util.context.Context context) {
-        Map<Object, Object> keyValues = context.stream().collect(Collectors.toMap(Entry::getKey, Entry::getValue));
-        if (ImplUtils.isNullOrEmpty(keyValues)) {
+        if (context == null) {
```

Comment:
This check is not needed. Reactor guarantees that `subscriberContext()` will return a non-null context.

Reply:
Removed